packages/@react-spectrum/breadcrumbs/docs/Breadcrumbs.mdx

```diff
@@ -0,0 +1,182 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/breadcrumbs';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Breadcrumbs} from '@react-spectrum/breadcrumbs';
+import {Item} from '@react-stately/collections';
+import {View} from '@react-spectrum/view';
+```
+
+# Breadcrumbs
+
+<p>{docs.exports.Breadcrumbs.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Breadcrumbs']} />
+
+## Example
+
+```tsx example
+<Breadcrumbs>
+  <Item uniqueKey="Folder 1">Folder 1</Item>
+  <Item uniqueKey="Folder 2">Folder 2</Item>
+  <Item uniqueKey="Folder 3">Folder 3</Item>
+</Breadcrumbs>
+```
+
+## Content
+
+Breadcrumbs accept as list of `children`, each with a `uniqueKey` prop.
+
+It is required the `children` are Item components from the `@react-stately/collections` library.
+
+```tsx example
+function Example() {
+  return (
+    <Breadcrumbs>
+      <Item uniqueKey="Folder 1">Folder 1</Item>
+      <Item uniqueKey="Folder 2">Folder 2</Item>
+      <Item uniqueKey="Folder 3">Folder 3</Item>
+    </Breadcrumbs>
+  );
+}
+```
+
+In order to identify children when a user takes an action on a item,
+each child needs to have a `uniqueKey` prop. See [Events](#events) for more information.
+
+### Internationalization
+
+In order to internationalize Breadcrumbs, the strings of all child items should be localized.
+
+## Events
+
+Use the `onAction` prop as a callback to handle press events on items.
+The `uniqueKey` prop from the selected item will be passed into the callback.
+
+```tsx example
+function Example() {
+  let folders = [
+    {id: 1, label: 'Folder 1'},
+    {id: 2, label: 'Folder 2'},
+    {id: 3, label: 'Folder 3'}
+  ];
+  let [folderId, setFolderId] = React.useState();
+  return (
+    <div>
+      <Breadcrumbs onAction={(a) => setFolderId(a)}>
+        {folders.map(f => <Item key={f.id} uniqueKey={f.id}>{f.label}</Item>)}
+      </Breadcrumbs>
+      <p>Last Selected Folder: {folderId}</p>
+    </div>
+  );
+}
+```
+
+## Props
+
+<PropTable component={docs.exports.Breadcrumbs} links={docs.links} />
+
+## Visual Options
+
+### Size
+
+The small size is useful to maintain Breadcrumbs while reducing the height required to give users context.
```

Comment:
Not sure I quite understand this sentence, is it basically saying use small if you need Breadcrumbs to be visible but vertical space is limited?

How about: "The small size is useful when available space is limited."